config/samples/vsphere-validator-compute.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: validation.spectrocloud.labs/v1alpha1
+kind: VsphereValidator
+metadata:
+  labels:
+    app.kubernetes.io/name: vspherevalidator
+    app.kubernetes.io/instance: vspherevalidator-sample
+    app.kubernetes.io/part-of: validator-plugin-vsphere
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/created-by: validator-plugin-vsphere
+  name: vspherevalidator-compute
+  namespace: validator
+spec:
+  datacenter: "Datacenter"
+  computeResourceRules:
+   - name: "rp-cluster2-palette-advanced-projects check resources"
+     clusterName: Cluster2
+     scope: resourcepool
+     entityName: "rp-cluster2-palette-advanced-projects"
+     nodepoolResourceRequirements:
+       - name: master-pool
```

Comment:
Please update to "control-plane-pool". The industry is trying to expunge master/slave from our collective vernacular.